hopsworks-api/src/main/webapp/WEB-INF/web.xml

```diff
@@ -51,6 +51,27 @@
     <servlet-name>GrafanaProxyServlet</servlet-name>
     <url-pattern>/grafana</url-pattern>
   </servlet-mapping>
+  <!-- tensorboard proxy servlet -->
+  <servlet>
+    <servlet-name>TensorboardProxyServlet</servlet-name>
+    <servlet-class>io.hops.hopsworks.api.tensorflow.TensorboardProxyServlet</servlet-class>
+    <init-param>
+      <param-name>targetUri</param-name>
+      <param-value>http://localhost:80/</param-value>
```

Comment:
This value does not matter here, I put it because I had to put something, but it is overwritten by the actual tensorboard address during execution.